crates/context/config/src/client/env/proxy/query.rs

```diff
@@ -73,4 +77,20 @@ impl<'a, T: Transport> ContextProxyQuery<'a, T> {
 
         utils::send(&self.client, Operation::Read(params)).await
     }
+
+    pub async fn get_context_value(&self, key: Vec<u8>) -> Result<Vec<u8>, ClientError<T>> {
+        let params = ContextVariableRequest { key };
+
+        utils::send(&self.client, Operation::Read(params)).await
+    }
+
+    pub async fn get_context_storage_entries(
+        &self,
+        offset: usize,
+        limit: usize,
+    ) -> Result<Vec<ContextStorageEntry>, ClientError<T>> {
+        let params = ContextStorageEntriesRequest { offset, limit };
```

Comment:
In descriptio you mentioned size but here I can see only offset and limit?

Reply:
updated description